### ISSUE TITLE
Insufficent description of how to install and setup medusa-plugin-auth

### ISSUE THREAD

**Issue by codeustad** [adrien2p/medusa-plugins#169]
Hello.

I tried to install and use medusa-plugin-auth with Auth0 and Firebase, but in both cases I did not achieve any effect what so ever running Medusa. It just continued using the jwt authentication built in to medusa, although I did exactly what was described in the documentation.

Can anybody describe in detail, what should be done to use Auth0 and/or Firebase with Medusa 1.20.x?

Thanks in advace!

**Comment by adrien2p** [comment]
The plugin only act as a man in the middle and delegate de validation/authorisation to the third part. Once validated we generate a token that medusa can consume. Does that answer your question ?

**Comment by codeustad** [comment]
> The plugin only act as a man in the middle and delegate de validation/authorisation to the third part. Once validated we generate a token that medusa can consume. Does that answer your question ?

Thanks for the answer. I get the idea of how external IDP works, but I still don't know how to set up e.g. Auth0 or Firebase correctly with Medusa and this plugin.

How is it supposed to work if the integration was correct: Will the login page redirect to Auth0/Firebase, or will I get the Medusa standard login page but in the background it will call Auth0/Firebase? Are there any working examples or videos/posts I could consider?

Thanks in advance!

**Comment by WFL-Niek** [comment]
I agree with OP, setup documentation is great but the follow-up could do with some more examples. Do new customers registered via the `/store/customers` endpoint automatically end up in the Auth0 environment, or do we have to register customers via Auth0?
